notifique/src/main/java/com/nathanrassi/notifique/NotifiqueListView.kt

```diff
@@ -95,6 +95,14 @@ internal class NotifiqueListView(
     }
   }
 
+  fun deselectSelected() {
+    GlobalScope.launch {
+      withContext(Dispatchers.Main) {
```

Comment:
Unnecessary. Don't do this. I posted the code to do it.
Remove the coroutines stuff from this method.

Also, maybe the method should be called deselectAll?